pallets/collator-allowlist/src/mock.rs

```diff
@@ -63,6 +63,7 @@ impl frame_system::Config for Test {
 	type SystemWeightInfo = ();
 	type SS58Prefix = ();
 	type OnSetCode = ();
+	type MaxConsumers = frame_support::traits::ConstU32<16>;
```

Comment:
Why ist this needed? What did Parity do? ^^

Reply:
https://github.com/paritytech/substrate/pull/10382